src/header.tsx

```diff
@@ -94,9 +96,17 @@ export const Header: React.FC<HeaderProps> = ({
                     <Stack space={8}>
                         {pretitle && renderRichText(pretitle, {color: vars.colors.textPrimary})}
                         {small ? (
-                            <Text4 regular role="heading" aria-level={2}>
+                            <Text
+                                mobileSize={textPresets.title2.size.mobile}
```

Comment:
why are you using title2 preset in header?

Reply:
It is requested in Jira's issue :)

Reply:
Discussed offline. We decided to use `Title2` and `Title3` for small and large header, respectively.